aries-site/src/data/color.js

```diff
@@ -243,6 +243,16 @@ const inputLight = [
     value: colors['validation-critical'].light,
     hex: colors['validation-critical'].light,
   },
+  {
+    name: 'validation-ok',
+    value: colors['validation-critical'].light,
```

Comment:
```suggestion
    value: colors['validation-ok'].light,
```
Change for all others too, same for `validation-warning`

Reply:
yes, I got an error that's why I left them as critical looks like the theme has to be merged first to call the colors.

Sorry, should have made a comment about that.